### ISSUE TITLE
Information

### ISSUE THREAD

**Issue by unnamed082** [akshaykadam771/SATELLITE-IMAGE-OBJECTS-RECOGNITION#1]
could you please inform:
1) how many hidden layers and epochs were used when training a neural network?
2) what is the learning algorithm (reverse error propagation?) and the activation function(Relu?) neurons were used?

**Comment by akshaykadam771** [comment]
The default Mask Rcnn model was used with default parameters without any hyperparameter tuning. The model was trained for around 100000 epochs which has a loss of 0.2824.
